charts/ovirt-csi-driver-4.20.0/templates/csi_driver_node.yaml

```diff
@@ -120,38 +121,35 @@ spec:
           ovirt_password: $OVIRT_PASSWORD
           # set a valid path only if ca bundle has content
           ovirt_cafile: ${OVIRT_CA_BUNDLE:+$OVIRT_CAFILE}
```

Comment:
Should this setting be controlled by the .Values.ovirt.caProvided flag that you added?

Reply:
This works for both secure and insecure (no ca) so I would prefer to leave it.